tests/System.Text.Primitives.Tests/Formatting/TimeTests.cs

```diff
@@ -51,6 +53,7 @@ public void RandomDateTimeTests(char format)
         static void TestDateTimeFormat(DateTime dt, char format, SymbolTable symbolTable)
         {
             var expected = dt.ToString(format.ToString(), CultureInfo.InvariantCulture);
+            if (format == 'r') expected = expected.ToLowerInvariant();
```

Comment:
Maybe, but we don't have UTF8 to lower conversion, so I cannot even try it. Moreover, I am not sure if all UTF8 to lower would be always non-resizing, so implementing it as an in-place conversion might not be possible.